tests/conftest.py

```diff
@@ -36,7 +47,42 @@ def pytest_internalerror(excinfo):
         raise excinfo.value
 
 
+def get_model_system(
+    type: str = 'original',
+    positions: List[List[float]] = [[0, 0, 0], [0.5, 0.5, 0.5]],
+    chemical_symbols: List[str] = ['Ga', 'As'],
+    orbital_symbols: List[List[str]] = [['s'], ['px', 'py']],
+) -> ModelSystem:
+    """
+    Create a `ModelSystem` object with the given parameters.
+    """
+    model_system = ModelSystem()
+    atomic_cell = AtomicCell(type=type, positions=positions * ureg.meter)
+    model_system.cell.append(atomic_cell)
+
+    # Add atoms_state to the model_system
+    atoms_state = []
+    for index, atom in enumerate(chemical_symbols):
+        orbitals = orbital_symbols[index]
+        orbitals_state = []
+        for orbital in orbitals:
+            orbitals_state.append(
+                OrbitalsState(
+                    l_quantum_symbol=orbital[0], ml_quantum_symbol=orbital[1:]
+                )
+            )
+        atom_state = AtomsState(chemical_symbol=atom, orbitals_state=orbitals_state)
+        # and obtain the atomic number for each AtomsState
+        atom_state.resolve_chemical_symbol_and_number(logger)
+        atoms_state.append(atom_state)
+    atomic_cell.atoms_state = atoms_state
+    return model_system
+
+
 def get_scf_electronic_band_gap_template(threshold_change: float = 1e-3) -> SCFOutputs:
```

Comment:
Just a minor comment: I'd use `get`/`set` rather for retrieving/storing data from a state.
Here, you're more so _generating_, _creating_, or _setting up_ from scratch.

Reply:
Yeah, changed it for `generate_`.